test/integration/test.js

```diff
@@ -0,0 +1,79 @@
+var assert = require('assert');
+var server = require('blueoak-server');
+var fs = require('fs');
+var testUtil = server.testUtility();
+global.__appDir = __dirname;
+
+/**
+ * You MUST have couchdb running locally with a database called 'basic' in order for these tests to work
+ */
```

Comment:
are there other requirements for these tests to work?
maybe that the DB have no security?

I think it would be good to put the info in a dedicated README.md file in the test/integration directory